version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.5-alpha",
+  "version": "2.6-alpha",
```

Comment:
Should this still be alpha?

Reply:
Yes, current version will be 2.5 and all new fixes/features will come under 2.6-alpha